appendices.tex

```diff
@@ -370,6 +370,152 @@ \chapter{Listing of application roles}
+	\item[tempdir] Directory to store temporary artifacts. E.g. PDF files are stored here before
+	being sent to the browser. The default on unix variants is /tmp/ledgersmb.
+	\item[cache\_templates] Determines if HTML templates used for the LedgerSMB UI will be stored
+	in precompiled form for improved performance. The value 0 (zero) means no caching; the value
+	1 means caching in the directory \$(tempdir)/lsmb\_templates; systems running multiple
+	LedgerSMB instances with customized UIs should use separate values for {\tt tempdir} for that
+	reason
+	\item[language] @@@TODO
+	\item[log\_level] @@@TODO
+	\item[DBI\_TRACE] @@@TODO
+	\item[pathsep] @@@TODO
+	\item[latex] Identifies if the system has \LaTeX available: 0 means no \LaTeX, 1 means \LaTeX supported
+	\item[check\_max\_invoices] @@@TODO
+	\item[max\_post\_size] States the maximum size in bytes of the body in case of a POST request; this
+	setting reduces the impact of DoS requests, if the system is under attack. The setting also limits the
+	maximum size of attachments (see \secref{sec-workflows-quotations-file-attachments}) that can be uploaded
+	\item[decimal\_places] @@@TODO
+	\item[cookie\_name] @@@TODO
+	\item[no\_db\_str] @@@TODO
+	
+\end{description}
+
+\subsubsection{'environment' section}
+\label{subsubsec-global-config-ledgersmb-conf-environment}
+
+\begin{description}[style=nextline]
+	\item[PATH] @@@TODO
+\end{description}
+
+
+\subsubsection{'paths' section}
+\label{subsubsec-global-config-ledgersmb-conf-paths}
+
+\begin{description}[style=nextline]
+	\item [spool] @@@TODO
+	\item [userspath] @@@TODO
+	\item [templates] @@@TODO
+	\item [images] @@@TODO
+	\item [memberfile] @@@TODO
+	\item [localepath] @@@TODO
+\end{description}
+
+\subsubsection{'programs' section}
+\label{subsubsec-global-config-ledgersmb-conf-programs}
+
+\begin{description}[style=nextline]
+	\item [gzip] @@@TODO
+\end{description}
+
+
+\subsubsection{'mail' section}
+\label{subsubsec-global-config-ledgersmb-conf-mail}
+
+\begin{description}[style=nextline]
+	\item [sendmail] @@@TODO
+	\item [smtphost] @@@TODO
+	\item [smtptimeout] @@@TODO
+	\item [backup\_email\_from] @@@TODO
+\end{description}
+
+\subsubsection{'printers' section}
+\label{subsubsec-global-config-ledgersmb-conf-printers}
+
+This section contains a list of printers and the commands
+to be executed in order for the output to be sent to the given printer with
+the document to be printed fed to the program through standard input.
+The example below shows how to send standard input data to a printer called ``laser''
+when selecting the item ``Laser'' in the LedgerSMB printer selection drop down.
+
+\texttt{
+	Laser    = lpr -Plaser
+}
+
+
+\subsubsection{'database' section}
+\label{subsubsec-global-config-ledgersmb-conf-database}
+
+\begin{description}[style=nextline]
+	\item [host] Name of the host to connect to. See the documentation of the {\tt -h} command line option at 
+	\url{http://www.postgresql.org/docs/9.2/static/app-psql.html}

```

Comment:
I know this is moved from elsewhere. I'm proposing to change it to point to the current documentation though: 

https://www.postgresql.org/docs/current/app-psql.html

Reply:
Already done.